### ISSUE TITLE
BEAT2 Gesture dimensions in the dataset

### ISSUE THREAD

**Issue by kruaxdk** [PantoMatrix/PantoMatrix#153]
Hello, I would like to know why the gestures in the BEAT2 dataset are represented in 165 dimensions, but in the code we have to convert 6D to 330 dimensions.In other words, gestures have 330 dimensions as the input dimension of the model, not 165。
  Looking forward to your reply, thank you

**Comment by kruaxdk** [comment]
At the same time, I would like to know whether modal tran and foot contact labels are necessary, and . Can I remove these two modes from your code below?
        tar_pose_lower = torch.cat([tar_pose_leg, tar_trans, tar_contact], dim=2)
        latent_all = torch.cat([tar_pose_6d, tar_trans, tar_contact], dim=-1)

**Comment by H-Liu1997** [comment]
hi, 

rot6d will help the convergence of the network, 
and the foot contact has postive impact on the lower body VQVAE,
